python/pyarrow/tests/parquet/test_pandas.py

```diff
@@ -269,18 +268,6 @@ def test_pandas_parquet_configuration_options(tempdir):
         tm.assert_frame_equal(df, df_read)
 
 
-@pytest.mark.pandas
-@pytest.mark.filterwarnings("ignore:Parquet format '2.0':FutureWarning")
-def test_spark_flavor_preserves_pandas_metadata():
-    df = _test_dataframe(size=100)
-    df.index = np.arange(0, 10 * len(df), 10)
-    df.index.name = 'foo'
-
-    result = _roundtrip_pandas_dataframe(df, {'version': '2.0',
-                                              'flavor': 'spark'})
```

Comment:
what happens on this test if we use a different parquet version?
I am wondering if this should still pass for `2.6`

Reply:
You are right, this and the bellow test should pass for newer writer versions and they do. I will remove 'version' arg altogether as we should probably be testing this on the default version of the writer.

Thanks for the check!